adal/build.gradle

```diff
@@ -46,6 +46,8 @@ android {
             manifest.srcFile 'src/main/AndroidManifest.xml'
             java.srcDirs = ['src/main/java', 'src/main/aidl']
         }
+
+        androidTest.setRoot('src/androidTest')
```

Comment:
I know. With the current Eclipse based environment setup, you have to manually specify the source sets.

If you use the maven/intellij way, you wouldnt have to do this.